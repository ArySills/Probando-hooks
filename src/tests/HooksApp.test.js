import { shallow } from "enzyme";
import { HooksApp } from "../HooksApp";

describe('Pruebas sobre HooksApp', () => {
    
    test('debe renderizar el componente <HooksApp /> correctamente', () => {
      
        const wrapper = shallow ( <HooksApp />)

        expect (wrapper).toMatchSnapshot();


    });
    
});

