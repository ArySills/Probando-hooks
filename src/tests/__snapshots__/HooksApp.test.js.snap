// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre HooksApp debe renderizar el componente <HooksApp /> correctamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
